library/pod-security-policy/apparmor/template.yaml

```diff
@@ -15,6 +17,7 @@ spec:
           type: object
           properties:
             allowedProfiles:
+              description: "An array of AppArmor profiles. Examples: `runtime/default`, `unconfined`."
```

Comment:
+1, though maybe add that to the template description instead of the field description?

Reply:
Agreed with @maxsmythe on full template description.

New version:

```yaml
    description: >-
      Configures an allow-list of AppArmor profiles for use by containers.
      This corresponds to specific annotations applied to a PodSecurityPolicy.
      For information on AppArmor, see
      https://kubernetes.io/docs/tutorials/clusters/apparmor/
```

LMK if anyone dislikes otherwise will resolve

Reply:
SG